src/game/Pet.h

```diff
@@ -143,8 +143,8 @@ extern const uint32 LevelStartLoyalty[6];
 
 #define ACTIVE_SPELLS_MAX           4
 
-#define PET_FOLLOW_DIST  1.0f
-#define PET_FOLLOW_ANGLE (M_PI_F/2.0f)
+#define PET_FOLLOW_DIST  2.00f
```

Comment:
If my evil children allow some free time, I'll test this in-game tonight or maybe tomorrow evening.
I should be able to give feedback of NPC's pets behaviour, along with warlock's pets. However, I was never much of a hunter player (that, is an understatement), so I will only be able to give "visual" feedback on the changes for the hunter's pets and it would be nice if someone else could backup me on those.